source/binding/CallExpression.cpp

```diff
@@ -301,7 +301,7 @@ Expression& CallExpression::fromArgs(Compilation& compilation, const Subroutine&
     if (bad)
         return badExpr(compilation, result);
 
-    if (context.flags.has(BindFlags::FunctionOrFinal) &&
+    if ((context.flags.has(BindFlags::Function) || context.flags.has(BindFlags::Final)) &&
```

Comment:
Should be able to write this a bit more succinctly as `context.flags.has(BindFlags::Function | BindFlags::Final)` (here and elsewhere in this diff)